<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MasterFrame</class>
 <widget class="QFrame" name="MasterFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>848</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame#MasterFrame {
	background: transparent;
}

QLabel#l_bg {
	background-image: url(&quot;:/img/bg.png&quot;);
}

QLabel#l_frame {
	background-image: url(&quot;:/img/frame.png&quot;)
}

QPushButton {
	border: none;
	background: transparent;
}

QPushButton:hover {
	background: transparent;
}

QPushButton#btn_exit {
    border-image: url(:/img/btn/close.png);
    background-repeat: no-repeat;
}

QPushButton#btn_exit:hover {
    border-image: url(:/img/btn/close_hover.png);
    background-repeat: no-repeat;
}

QPushButton#btn_exit:pressed  {
    border-image: url(:/img/btn/close_pressed.png);
    background-repeat: no-repeat;
}

QPushButton#btn_minimize {
    border-image: url(:/img/btn/minimize.png);
    background-repeat: no-repeat;
}

QPushButton#btn_minimize:hover {
    border-image: url(:/img/btn/minimize_hover.png);
    background-repeat: no-repeat;
}

QPushButton#btn_minimize:pressed  {
    border-image: url(:/img/btn/minimize_pressed.png);
    background-repeat: no-repeat;
}

QPushButton#btn_settings {
    border-image: url(:/img/btn/settings.png);
    background-repeat: no-repeat;
}

QPushButton#btn_settings:hover {
    border-image: url(:/img/btn/settings_hover.png);
    background-repeat: no-repeat;
}

QPushButton#btn_settings:pressed  {
    border-image: url(:/img/btn/settings_pressed.png);
    background-repeat: no-repeat;
}

QPushButton#btn_repair {
    border-image: url(:/img/btn/repair.png);
    background-repeat: no-repeat;
}

QPushButton#btn_repair:hover {
    border-image: url(:/img/btn/repair_hover.png);
    background-repeat: no-repeat;
}

QPushButton#btn_repair:pressed  {
    border-image: url(:/img/btn/repair_pressed.png);
    background-repeat: no-repeat;
}

QPushButton#btn_ticket {
    border-image: url(:/img/btn/ticket.png);
    background-repeat: no-repeat;
}

QPushButton#btn_ticket:hover {
    border-image: url(:/img/btn/ticket_hover.png);
    background-repeat: no-repeat;
}

QPushButton#btn_ticket:pressed  {
    border-image: url(:/img/btn/ticket_pressed.png);
    background-repeat: no-repeat;
}

QLabel {
	color: #4F1717;
}

QLabel#l_version {
		color: #E4E8EA;
}

QWidget#menu_widget {
	background-image: url(&quot;:img/menu.png&quot;);
	background-repeat: no-repeat;
}

QWidget#login_widget {
	background-image: url(&quot;:img/login.png&quot;);
	background-repeat: no-repeat;
}
</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::Shape::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Shadow::Raised</enum>
  </property>
  <widget class="QPushButton" name="btn_exit">
   <property name="geometry">
    <rect>
     <x>1318</x>
     <y>40</y>
     <width>36</width>
     <height>36</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="iconSize">
    <size>
     <width>26</width>
     <height>26</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_settings">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>740</y>
     <width>48</width>
     <height>48</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="iconSize">
    <size>
     <width>72</width>
     <height>51</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="alicia1">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>500</y>
     <width>102</width>
     <height>105</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="icon">
    <iconset resource="master.qrc">
     <normaloff>:/tmp/game_selected_1.png</normaloff>:/tmp/game_selected_1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>102</width>
     <height>105</height>
    </size>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="alicia2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>620</y>
     <width>102</width>
     <height>105</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ForbiddenCursor</cursorShape>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="icon">
    <iconset resource="master.qrc">
     <normaloff>:/img/game_select_locked.png</normaloff>:/img/game_select_locked.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>102</width>
     <height>105</height>
    </size>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_minimize">
   <property name="geometry">
    <rect>
     <x>1268</x>
     <y>39</y>
     <width>36</width>
     <height>36</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="iconSize">
    <size>
     <width>26</width>
     <height>26</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="l_frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>848</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="content">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>840</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="l_bg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1400</width>
      <height>848</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="l_corrupted">
    <property name="geometry">
     <rect>
      <x>769</x>
      <y>595</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NanumGothic</family>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <hintingpreference>PreferNoHinting</hintingpreference>
      <fontweight>DemiBold</fontweight>
     </font>
    </property>
    <property name="text">
     <string>Corrupted files?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_repair">
    <property name="geometry">
     <rect>
      <x>769</x>
      <y>623</y>
      <width>160</width>
      <height>32</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>160</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ticket">
    <property name="geometry">
     <rect>
      <x>1017</x>
      <y>623</y>
      <width>160</width>
      <height>32</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>160</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="login_widget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>715</x>
      <y>240</y>
      <width>516</width>
      <height>292</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="l_issues">
    <property name="geometry">
     <rect>
      <x>1017</x>
      <y>595</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NanumGothic</family>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <hintingpreference>PreferNoHinting</hintingpreference>
      <fontweight>DemiBold</fontweight>
     </font>
    </property>
    <property name="text">
     <string>Ran into issues?</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_version">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1127</x>
      <y>762</y>
      <width>211</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>EurostileBla</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="text">
     <string>GAME VERSION 1.0.0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="l_game_start_frame">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>266</x>
      <y>383</y>
      <width>365</width>
      <height>298</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="master.qrc">:/img/game_start_frame.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="l_game_start">
    <property name="geometry">
     <rect>
      <x>266</x>
      <y>383</y>
      <width>365</width>
      <height>298</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="menu_widget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>715</x>
      <y>240</y>
      <width>516</width>
      <height>292</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="l_troubleshoot">
    <property name="geometry">
     <rect>
      <x>769</x>
      <y>568</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>EurostileBla</family>
      <pointsize>14</pointsize>
      <hintingpreference>PreferNoHinting</hintingpreference>
      <fontweight>Black</fontweight>
     </font>
    </property>
    <property name="text">
     <string notr="true">TROUBLESHOOT</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_support">
    <property name="geometry">
     <rect>
      <x>1017</x>
      <y>568</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>EurostileBla</family>
      <pointsize>14</pointsize>
      <hintingpreference>PreferNoHinting</hintingpreference>
      <fontweight>Black</fontweight>
     </font>
    </property>
    <property name="text">
     <string notr="true">SUPPORT</string>
    </property>
   </widget>
   <zorder>l_bg</zorder>
   <zorder>l_corrupted</zorder>
   <zorder>btn_repair</zorder>
   <zorder>btn_ticket</zorder>
   <zorder>login_widget</zorder>
   <zorder>l_issues</zorder>
   <zorder>l_version</zorder>
   <zorder>l_game_start</zorder>
   <zorder>menu_widget</zorder>
   <zorder>l_troubleshoot</zorder>
   <zorder>l_support</zorder>
   <zorder>l_game_start_frame</zorder>
  </widget>
  <zorder>content</zorder>
  <zorder>l_frame</zorder>
  <zorder>btn_minimize</zorder>
  <zorder>btn_exit</zorder>
  <zorder>btn_settings</zorder>
  <zorder>alicia2</zorder>
  <zorder>alicia1</zorder>
 </widget>
 <resources>
  <include location="master.qrc"/>
 </resources>
 <connections/>
</ui>
